adapters/rubicon/rubicon.go

```diff
@@ -693,8 +694,14 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adap
 				continue
 			}
 
+			// if imp.ext.is_rewarded_inventory = 1, set imp.video.ext.videotype = "rewarded"
+			var videoType = ""
+			if bidderExt.Prebid != nil && bidderExt.Prebid.IsRewardedInventory == 1 {
+				videoType = "rewarded"
+			}
+
 			videoCopy := *thisImp.Video
-			videoExt := rubiconVideoExt{Skip: rubiconExt.Video.Skip, SkipDelay: rubiconExt.Video.SkipDelay, RP: rubiconVideoExtRP{SizeID: rubiconExt.Video.VideoSizeID}}
+			videoExt := rubiconVideoExt{Skip: rubiconExt.Video.Skip, SkipDelay: rubiconExt.Video.SkipDelay, VideoType: videoType, RP: rubiconVideoExtRP{SizeID: rubiconExt.Video.VideoSizeID}}
 			videoCopy.Ext, err = json.Marshal(&videoExt)
```

Comment:
Do we need to give the recently added `VideoType` field a value also on the legacy side of this adapter? Or is this field only for the OpenRTB2 endpoints?
```
 364 func (a *RubiconAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
 365     callOneObjects := make([]callOneObject, 0, len(bidder.AdUnits))
 366 +-- 54 lines: supportedMediaTypes := []pbs.MediaType{pbs.MEDIA_TYPE_BANNER, pbs.MEDIA_TYPE_VIDEO}-------------------------------------------------------------------------
 420
 421         if thisImp.Video != nil {
 422             videoExt := rubiconVideoExt{Skip: params.Video.Skip, SkipDelay: params.Video.SkipDelay, RP: rubiconVideoExtRP{SizeID: params.Video.VideoSizeID}}
 423             thisImp.Video.Ext, err = json.Marshal(&videoExt)
 424         } else {
 425             primarySizeID, altSizeIDs, err := parseRubiconSizes(unit.Sizes)
adapters/rubicon/rubicon.go
```

Reply:
Don't think so. This feature refers only for OpenRTB, no legacy support.

Reply:
Got ya. Approved